resources/magento2/inspection.properties

```diff
@@ -1,2 +1,15 @@
 inspection.plugin.duplicateInSameFile=The plugin name already used in this file. For more details see Inspection Description.
-inspection.plugin.duplicateInOtherPlaces=The plugin name "{0}" for targeted "{1}" class is already used in the module "{2}" ({3} scope). For more details see Inspection Description.
\ No newline at end of file
+inspection.plugin.duplicateInOtherPlaces=The plugin name "{0}" for targeted "{1}" class is already used in the module "{2}" ({3} scope). For more details see Inspection Description.
+must.not.be.empty.should.contain.letters.or.numbers=Must not be empty, should contain letters or numbers
+package.name=Package Name
```

Comment:
I believe these messages aren't related to any of inspections.

Reply:
@VitaliyBoyko thank you for your review. I investigated the form edit functionality deeper and think updated PR should be better now)